src/ts/components/basic-components.ts

```diff
@@ -0,0 +1,69 @@
+import styled from 'styled-components'
+
+export const LayoutContainer = styled('div')`
+    width: 375px;
+    height: 667px;
+    border-radius: 4px;
+    box-shadow: 0 6px 30px 0 ${props => props.theme.shadowColor};
+    border: solid 1px ${props => props.theme.borderColor};
+    background-color: ${props => props.theme.backgroundColor};
+`
+
+export const Button = styled.button`
+  width: 311px;
+  height: 45px;
+  border-radius: 4px;
+  box-shadow: 0 3px 10px 0 rgba(72, 178, 228, 0.21);
+  background-color: #24b6e8;
+  font-family: Nunito;
+  font-size: 16px;
+  font-weight: 800;
+  font-style: normal;
+  font-stretch: normal;
+  line-height: 1.31;
+  letter-spacing: normal;
+  text-align: center;
+  color: #ffffff;
+`
+
+export const Section = styled.p`
```

Comment:
better to use `div` instead of `p` as section will contain other components

Reply:
agreed. updated and pushed